app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -56,95 +71,122 @@ fun HomeScreen(
     HomeScreenContent(
         showMyCoinsLoading = showMyCoinsLoading,
         showTrendingCoinsLoading = showTrendingCoinsLoading,
+        isRefreshing = rememberRefreshing,
         myCoins = myCoins,
         trendingCoins = trendingCoins,
         onMyCoinsItemClick = viewModel.input::onMyCoinsItemClick,
         onTrendingItemClick = viewModel.input::onTrendingCoinsItemClick
-    )
+    ) { viewModel.loadData(isRefreshing = true) }
 }
 
+@OptIn(ExperimentalMaterialApi::class)
 @Suppress("LongParameterList")
 @Composable
 private fun HomeScreenContent(
     showMyCoinsLoading: IsLoading,
     showTrendingCoinsLoading: IsLoading,
+    isRefreshing: IsLoading,
     myCoins: List<CoinItemUiModel>,
     trendingCoins: List<CoinItemUiModel>,
     onMyCoinsItemClick: (CoinItemUiModel) -> Unit,
-    onTrendingItemClick: (CoinItemUiModel) -> Unit
+    onTrendingItemClick: (CoinItemUiModel) -> Unit,
+    onRefreshing: () -> Unit
```

Comment:
I think the event name should be in the present tense, when comparing to `onTrendingItemClick` or `onLoadMore` in [my PR](https://github.com/nimblehq/jetpack-compose-crypto/pull/62).

```suggestion
    onRefresh: () -> Unit
```